utils/file_utils.jsx

```diff
@@ -35,3 +35,31 @@ export function trimFilename(filename) {
 
     return trimmedFilename;
 }
+
+export function getFileDimentionsForDisplay(dimentions, {maxHeight, maxWidth}) {
+    const {width, height} = dimentions;
+    if (height <= maxHeight && width <= maxWidth) {
+        return dimentions;
+    }
+    const widthRatio = width / maxWidth;
+    const heightRatio = height / maxHeight;
+    if (heightRatio > widthRatio) {
+        return {
+            height: maxHeight,
+            width: width * (1 / heightRatio),
+        };
+    }
+
+    return {
+        height: height * (1 / widthRatio),
+        width: maxWidth,
+    };
+}
+
+export function getOpenGraphMetadataByLink(embedData, url) {
```

Comment:
actually I have the same function in RN , maybe we can move this helper function to `mattermost-redux` instead of having it in both projects? 0/5